src/styles/components/chat/_chat-prompt-wrapper.scss

```diff
@@ -93,13 +95,11 @@
                 }
 
                 > .mynah-chat-prompt-input-sizer {
-                    display: block;
+                    display: inline-block;
                     width: 100%;
                     opacity: 1;
                     pointer-events: none;
-                    overflow: hidden;
-                    white-space: pre-wrap;
-                    word-break: break-word;
+                    overflow: auto;
```

Comment:
This one makes the span which is being used to represent the text (textarea is invisible) render properly with scroll changes.